LICENSE

```diff
@@ -254,13 +263,19 @@ https://github.com/elastic/elasticsearch-cloud-aws
 developed by Elastic
 License: Apache License v2 (http://www.apache.org/licenses)
 
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
```

Comment:
@mxm

Reply:
Also part of ES now.

Reply:
@mxm so not a part of our source? (where "source" here means everything you check out by default not including build time downloads and such)

Reply:
It is part of `es/upstream`.

Reply:
ok thanks!